gtsfm/data_association/point3d_initializer.py

```diff
@@ -85,14 +120,15 @@ def execute_ransac_variant(self, track_2d: SfmTrack2d) -> np.ndarray:
             best_inliers: boolean array of length N. Indices of measurements
                are set to true if they correspond to the best RANSAC hypothesis
         """
-        if self.num_ransac_hypotheses is None:
-            raise ValueError("RANSAC triangulation requested but number of hypothesis is None.")
+        # if self.num_ransac_hypotheses is None:
```

Comment:
just curious -- why do we not need this check anymore?

Reply:
Because the checks are performed by `__check_ransac_params()` when `num_ransac_hypotheses()` is called.